### PR TITLE
DOCS: Update the description of the fused parameter.

### PR DESCRIPTION
**Pull request by codrut3**
Update the description of layers.batch_normalization and contrib.layers.python.layers.batch_norm to reflect the fact that fused=None is equivalent to fused=True. In fact, both functions check at the beginning if fused is None, and if yes they set it to True.

### CODE REVIEW

**Comment by martinwicke** [comment]
This change is good -- the docs were clearly out of date. 

However, the actual behavior of batch_norm is a bit puzzling to me: If fused is None or True, it tries to use fused batch norm. I would have expected 

fused=None: Try fused, fall back on regular if not possible to use fused
fused=True: Try fused, if not possible, raise
fused=False: Don't try fused.

@chrisying, do you have context on this (just calling you because you're in the comments)

**Comment by chrisying** [comment]
The current behavior is that fused=None and fused=True have the same behavior (try to use fused=True but revert to fused=False if the conditions are not met). I'm not entirely sure why this but might possibly be due to backwards compatibility? Not certain.

tf.layers.batch_normalization should be cleaned-up in the future since there are quite a few optional parameters right now, many of which are not compatible. But that's a separate issue, this doc change is technically correct but I can't confirm if it's intended or not.

**Review by martinwicke — APPROVED** [review]
